Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/FocusHandlerTrigger.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Utility component to hook up Unity events to the OnFocusEnter and OnFocusExit events.
+    /// </summary>
+    public class FocusHandlerTrigger : MonoBehaviour, IMixedRealityFocusHandler
+    {
+        /// <summary>
+        /// Event which is triggered when focus begins.
+        /// </summary>
+        [Header("Events")]
+        public UnityEvent OnFocusEnterEvent;
+
+        /// <summary>
+        /// Event which is triggered when focus ends.
+        /// </summary>
+        public UnityEvent OnFocusExitEvent;
+
+        /// <inheritdoc />
+        public void OnFocusEnter(FocusEventData eventData)
+        {
+            OnFocusEnterEvent.Invoke();
```

Comment:
I think may want to check for OnFocusEnterEvent?.Invoke()?

Reply:
We shouldn't need to since Unity's serialization system will initialize all serializable properties for us. But, I suppose a user could nullify the event? I can make the change.